h_periodic/h_beat.py

```diff
@@ -17,8 +17,8 @@
     print("h_beat disabled by DISABLE_H_BEAT environment variable")
     sys.exit()
 
-SLIM_TASK_SINCE = "2018-01-01"
```

Comment:
We don't order the candidate annotations (because it's expensive) so these are not guaranteed to be processed in order although they have been until now.


The oldest non-migrated annotations is from April:

```
select annotation.created from annotation 
left outer join annotation_slim on annotation_slim.pubid = annotation.id
where pubid is null
order by annotation.created asc
limit 1
```